shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -24,24 +24,43 @@
   {{else}}
     {{!-- Becomes hamburger menu button in mobile view. --}}
     <div class="menu-button"></div>
-
     <ul class="topbar {{#if _menuExpanded.get}}expanded{{/if}}">
-      {{#each items}}
-        {{#unless onlyPopup}}
-          <li class="{{name}}">
-            {{#if ./popupTemplate}}
-              <button class="show-popup">
+      <ul class="menubar">
+        {{#each items}}
+          {{#unless onlyPopup}}
+            <li class="{{name}}">
+              {{#if ./popupTemplate}}
+                <button class="show-popup">
+                  {{> template data.get}}
+                </button>
+              {{else}}
                 {{> template data.get}}
-              </button>
-            {{else}}
-              {{> template data.get}}
-            {{/if}}
+              {{/if}}
+            </li>
+          {{/unless}}
+        {{/each}}
+      </ul>
+      <ul class="navbar {{#unless _showSidebar.get}}hide-desktop{{/unless}}">
+        <li class="navitem-open-grain {{#if currentRoute 'selectGrain'}}current{{/if}}"><a href="{{pathFor route='selectGrain'}}"><span>Open...</span></a></li>
+        <li class="navitem-create-grain {{#if currentRoute 'newGrain'}}current{{/if}}"><a href="{{pathFor route='newGrain'}}">New...</a></li>
```

Comment:
Good catch.

Do we actually need a topbar at all for the debug log?  I'm inclined to remove the topbar from lightLayout entirely, or maybe replace it with a single line with the grain title.

(my usual experience with the debug log is that I only look at it in dev mode, and then I wind up opening a new one every time I make a new grain, and then I have like 15 indistinguishable dev mode debug log windows)